extensions/mapstore/pom.xml

```diff
@@ -20,13 +20,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.hazelcast</groupId>
```

Comment:
Why do we need it? Other extensions declare only `artifactId`

Reply:
The `groupId` is inherited from the parent, which is `com.hazelcast.jet` for the extensions. If we want to have `com.hazelcast` for the `groupId` we need to override it here.

Maybe I misunderstood what you wanted to do here https://github.com/hazelcast/hazelcast/pull/22213#discussion_r970704102. Can you clarify that? :-)

Reply:
ah, all right, you got me right :)